cyvlfeat/hog/hog.py

```diff
@@ -82,14 +89,24 @@ def hog(image, cell_size, variant='UoCTTI', n_orientations=9,
         raise ValueError('Expected a polar field image of n_channels == 2')
 
     # Ensure types are correct before passing to Cython
-    image = np.require(image, dtype=np.float32, requirements='F')
+    image = np.require(image, dtype=np.float32, requirements='C')
```

Comment:
This is probably what's breaking the tests

Reply:
I don't think so. According to vlfeat's doc, it uses row-major for image. see also https://www.vlfeat.org/api/conventions.html. I'm not quite sure how it deal with color images.

Reply:
I think it's fixed now. VLfeat uses channel-first style.